src/reflect/mima-filters/2.12.0.forwards.excludes

```diff
@@ -8,6 +8,7 @@ ProblemFilters.exclude[MissingClassProblem]("scala.reflect.io.PlainNioFile")
 
 ProblemFilters.exclude[DirectMissingMethodProblem]("scala.reflect.runtime.SynchronizedOps.newMappedBaseTypeSeq")
 ProblemFilters.exclude[DirectMissingMethodProblem]("scala.reflect.runtime.JavaUniverse.newMappedBaseTypeSeq")
+ProblemFilters.exclude[DirectMissingMethodProblem]("scala.reflect.runtime.JavaUniverse.statistics")
```

Comment:
Whitelisted because it's `protected`. Same case as `newMappedBaseTypeSeq` which is above.